src/organizations/components/OrgProfileTab/CopyableLabeledData.tsx

```diff
@@ -19,20 +22,35 @@ import {notify} from 'src/shared/actions/notifications'
 // Utils
 import {copyToClipboardSuccess} from 'src/shared/copy/notifications'
 import 'src/organizations/components/OrgProfileTab/style.scss'
+import {getOrg} from 'src/organizations/selectors'
 
 interface Props {
   id: string
   label: string
   src: string
   name?: string
+  renameableOrg?: boolean
 }
 
-const CopyableLabeledData: FC<Props> = ({id, label, src, name}) => {
+const CopyableLabeledData: FC<Props> = ({
+  id,
+  label,
+  src,
+  name,
+  renameableOrg,
```

Comment:
one suggestion on boolean variable naming:

When naming booleans, it generally helps remove ambiguity to have some kind of conjugation of the verb `to be`. `renameableOrg` could be thought of as a value, as in 
```ts
const renameableOrg = 'org1234,org567'
```
It might be read as the org that is renamable, rather than answering the question, is this org renameable.

Contrast that with `orgIsRenameable` or `isOrgRenameable` or even `isRenameableOrg`. The `is`, which is a conjugation of the verb `to be`, which indicates existence, hints that it's a boolean. It's kind of one of the only places where modern coding conventions use [Hungarian Notation.](https://en.wikipedia.org/wiki/Hungarian_notation)

So down below it would change to something like

```ts
{isOrgRenameable && (
  <Button
```

which is a small thing, but helps clue in the reader to the intent of the code.